src/main/kotlin/org/rust/lang/refactoring/RsDowngradeModuleToFile.kt

```diff
@@ -50,10 +51,10 @@ private fun contractModule(fileOrDirectory: com.intellij.psi.PsiFileSystemItem)
 
 private val com.intellij.psi.PsiElement.isDirectoryMod: Boolean get() {
     return when (this) {
-        is org.rust.lang.core.psi.RsFile -> name == org.rust.lang.core.psi.ext.RsMod.Companion.MOD_RS && containingDirectory?.children?.size == 1
+        is org.rust.lang.core.psi.RsFile -> name == RsConstants.MOD_RS_FILE && containingDirectory?.children?.size == 1
```

Comment:
Hmm, why we don't use imports here? If we anyway editing this line, maybe import it?

Reply:
done